lib/live_debugger/live_views/sidebar_live.ex

```diff
@@ -104,12 +112,49 @@ defmodule LiveDebugger.LiveViews.SidebarLive do
           node_id={@node_id}
           highlight?={@highlight?}
           parent_lv_process={@parent_lv_process}
+          memory_raw_kb={@memory_raw_kb}
+          memory_gc_kb={@memory_gc_kb}
+          assigns_memory_kb={@assigns_memory_kb}
         />
       </.sidebar_slide_over>
     </div>
     """
   end
 
+  @impl true
+  def handle_info(:assign_memory_usage, socket) do
+    Process.send_after(self(), :assign_memory_usage, 2000)
+
+    raw =
+      case :erlang.process_info(self(), :memory) do
+        {:memory, bytes} -> bytes
+        _ -> 0
+      end
+
+    :erlang.garbage_collect(self())
+
+    post_gc =
+      case :erlang.process_info(self(), :memory) do
+        {:memory, bytes} -> bytes
+        _ -> 0
+      end
+
+    assigns_size =
+      try do
+        words = :erts_debug.size(socket.assigns)
```

Comment:
As above - you are calculating here size of `SidebarLive` assigns

Reply:
Yep - thanks. Followed other example of using the ChannelService to pull in the node and then calculate the size on that assigns. @kraleppa